app/src/molecules/PipetteSelect/index.tsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react'
+import groupBy from 'lodash/groupBy'
+import { useTranslation } from 'react-i18next'
+import {
+  getAllPipetteNames,
+  getPipetteNameSpecs,
+  GEN1,
+  GEN2,
+  SINGLE_CHANNEL,
+  EIGHT_CHANNEL,
+} from '@opentrons/shared-data'
+import { Box, Flex } from '@opentrons/components'
+import { Select } from '../../atoms/SelectField/Select'
+
+import type { PipetteNameSpecs } from '@opentrons/shared-data'
+import type { ActionMeta, SingleValue, MultiValue } from 'react-select'
+import type { SelectOption } from '../../atoms/SelectField/Select'
+
+export interface PipetteSelectProps {
+  /** currently selected value, optional in case selecting triggers immediate action */
+  pipetteName?: string | null
+  /** react-select change handler */
+  onPipetteChange: (
+    pipetteName: string | null,
+    e: ActionMeta<SelectOption>
+  ) => void
+  /** list of pipette names to omit */
+  nameBlocklist?: string[]
+  /** whether or not "None" shows up as the default option */
+  enableNoneOption?: boolean
+  /** input tabIndex */
+  tabIndex?: string | number
```

Comment:
L82 is using `as number` and in this PR tabIndex isn't passed so we can use only `number`? 

```suggestion
  tabIndex?: number
```